osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -24,6 +29,32 @@ private void load(OsuColour colours)
             AddButton("Continue", colours.Green, () => OnResume?.Invoke());
             AddButton("Retry", colours.YellowDark, () => OnRetry?.Invoke());
             AddButton("Quit", new Color4(170, 27, 39, 255), () => OnQuit?.Invoke());
+
+            AddInternal(pauseLoop = new SkinnableSound(new SampleInfo("pause-loop"))
+            {
+                Looping = true,
+            });
+            // PopIn is called before updating the skin, and when a sample is updated, its "playing" value is reset
+            // the sample must be played again(and if it plays when it shouldn't, the volume will be at 0)
+            pauseLoop.OnSkinChanged += () => pauseLoop.Play();
```

Comment:
If the pause overlay isn't visible, this won't even fire (as the pause overlay is not present), so this seems to be even safer than written here.